### ISSUE TITLE
Too many arguments for function: strcharpart 

### ISSUE THREAD

**Issue by jam1015** [ggandor/flit.nvim#38]
Same issue as [https://github.com/ggandor/leap.nvim/issues/171](https://github.com/ggandor/leap.nvim/issues/171)

To get flit working I have to revert back to both of these commits:

```	
{

		"ggandor/flit.nvim",
		commit = "5c9a78b97f7f4301473ea5e37501b5b1d4da167b",
	},
	{
		"ggandor/leap.nvim",
		commit ="8facf2eb6a378fd7691dce8c8a7b2726823e2408",
	},
```

**Comment by jakubbortlik** [comment]
First of all, thanks for the nice plugin!
Can you please reopen the issue (or just fix the bug for good :)). The solution in commit `94822e3` does not work for me on nvim-0.9.1, because `vim.fn.has("nvim-0.10")` evaluates to `0` which in lua evaluates to `true` (unlike in, e.g., Python). So the correct solution must contain: `if skipcc == 1 then`, rather than just a bare `if skipcc then` (`flit.nvim/lua/flit.lua` line 67).

**Comment by ggandor** [comment]
Fixed by 498b3a4864e697f1ed1145e518c4c78c776c55d8.
